code/wp-plugin-litespeed-cache.md

```diff
@@ -0,0 +1,73 @@
+# Plugin WP - Litespeed Cache #
+
+## Dashboard ##
+Aucune modification d'options a faire ici.
+
+## General ##
+Lier a un quic.cloud account (fpilon@pixelcircus.ca; le mot de passe est dans le fichier d'accès.) Ce cloud account sera utilisé pour l'optimisation des images.
+
+## Cache ##
+
+### General ###
+Tout devrait être ON except cache mobile.
+
+Cache logged in users peut être a OFF pour tester lorsque on place du nouveau code en prod.
```

Comment:
Mettre "doit" ici. C'est chiant quand on ne sait pas comment le site est configuré. On doit normaliser au maximum.

"Cache logged in users doit être a OFF"